kernel/byterun/dune

```diff
@@ -4,7 +4,12 @@
  (public_name coq.vm)
  (c_names coq_fix_code coq_memory coq_values coq_interp))
 
+(rule
+ (targets coq_instruct.h)
+ (deps (:gen ../genOpcodeFiles.exe))
+ (action (run %{gen} "enum")))
+
 (rule
  (targets coq_jumptbl.h)
- (deps (:h-file coq_instruct.h) make_jumptbl.sh)
- (action (bash "./make_jumptbl.sh %{h-file} %{targets}")))
+ (deps (:gen ../genOpcodeFiles.exe))
+ (action (run %{gen} "jump")))
```

Comment:
You can also do this in a single rule if you want, also gen is not strictly needed:
```
(rule
  (targets coq_instruct.h coq_jumptbl.h)
  (action (run ../genOpcodefiles.exe jump enum))
```
or
```
(action (progn (run ../... ) (run ...))
```
You can also scope the binary using the binary scoping feature, this allows you to extend path so you don't have to use `../gen`...